dev/builder/build.sh

```
@@ -75,12 +75,13 @@ then
 	VERSION="$VERSION DEV"
 fi
 echo ""
+
 if [ "$UNMINIFIED" = true ]; then
 	echo "Building unminified version..."
-	java -jar ckbuilder/$CKBUILDER_VERSION/ckbuilder.jar --build ../../ release $JAVA_ARGS --version="$VERSION" --revision="$REVISION" --overwrite --leave-js-unminified --leave-css-unminified
+	java --add-exports java.desktop/sun.java2d=ALL-UNNAMED -jar ckbuilder/$CKBUILDER_VERSION/ckbuilder.jar --build ../../ release $JAVA_ARGS --version="$VERSION" --revision="$REVISION" --overwrite --leave-js-unminified --leave-css-unminified
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)